demos/twitter/src/main/java/com/datatorrent/demos/twitter/RSSFeedExtractor.java

```
@@ -0,0 +1,160 @@
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+/**
+ * <p>
+ * TwitterStatusUserExtractor class.
+ * </p>
+ *
+ * @since 0.3.2
+ */
+public class RSSFeedExtractor implements InputOperator
+{
+  protected String URLToConvert = "";
+  protected BufferedReader in;
+  URL rssURL;
+  public final transient DefaultOutputPort<String> output = new DefaultOutputPort<String>();
+
+  @Override
+  public void beginWindow(long windowId)
+  {
+    // TODO Auto-generated method stub
+  }
+
+  @Override
+  public void endWindow()
+  {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void setup(OperatorContext context)
+  {
+    try {
+      DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+      DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+      InputStream is = this.getClass().getClassLoader()
+          .getResourceAsStream("RSS.xml");
+      Document doc = null;
+      try {
+        doc = dBuilder.parse(is);
+      } catch (SAXException e) {
+        throw new RuntimeException("SAX Exception");
+      }
+
+      doc.getDocumentElement().normalize();
+
+      NodeList nList = doc.getElementsByTagName("sub");
+
+      for (int index = 0; index < nList.getLength(); index++) {
+        Node nNode = nList.item(index);
+        if (nNode.getNodeType() == Node.ELEMENT_NODE) {
+          Element eElement = (Element) nNode;
+          URLToConvert = eElement.getElementsByTagName("value").item(0)
+              .getTextContent();
+        }
+      }
+
+    } catch (ParserConfigurationException e) {
+      throw new RuntimeException("Parser Configuration Exception");
+
+    } catch (IOException e) {
+      throw new RuntimeException("Runtime Exception");
```

Can you also pass "e" as parameter to RuntimeException? Basically not masking real exception will helpful in case of any issues.

Also, can you please put a better message in RuntimeException?